playbooks/roles/openvpn/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ openvpn_clients:
 
 openvpn_port: "636"
 openvpn_port_udp: "8757"
+openvpn_port_sslh: "443"
 openvpn_key_size: "2048"
-openvpn_cipher: "AES-256-CBC"
+openvpn_cipher: "AES-256-GCM"
+openvpn_ncp_cipher: "AES-256-GCM:AES-256-CBC"
```

Comment:
However, this would only work if the client profiles for 2.3 clients specified a 2.3-compatible cipher. Right now they would have AES-256-GCM as their cipher, and 2.3 clients wouldn't know what to do with that.